deployer/conf/elasticsearch.yml

```diff
@@ -1,97 +1,60 @@
-
-discovery:
-  type: io.fabric8.elasticsearch.discovery.k8s.K8sDiscoveryModule
-
 path:
   data: /elasticsearch/persistent/${CLUSTER_NAME}/data
   logs: /elasticsearch/${CLUSTER_NAME}/logs
   work: /elasticsearch/${CLUSTER_NAME}/work
+  scripts: /elasticsearch/${CLUSTER_NAME}/scripts
 
 searchguard:
+  authcz.admin_dn:
+  - CN=system.admin,OU=client,O=client,L=Test,C=DE
   config_index_name: ".searchguard.${HOSTNAME}"
-  key_path: /elasticsearch/${CLUSTER_NAME}
-  allow_all_from_loopback: false
-  authentication:
-    authentication_backend:
-      impl: com.floragunn.searchguard.authentication.backend.simple.AlwaysSucceedAuthenticationBackend
-    authorizer:
-      impl: com.floragunn.searchguard.authorization.simple.SettingsBasedAuthorizator
-    http_authenticator:
-      impl: io.fabric8.elasticsearch.plugin.HTTPSProxyClientCertAuthenticator
-    proxy:
-      header: X-Proxy-Remote-User
-      trusted_ips: ["*"]
-    authorization:
-      settingsdb:
-        roles:
-          admin: ["admin"]
-          fluentd: ["fluentd"]
-          kibana: ["kibana"]
-          curator: ["curator"]
   ssl:
     transport:
-      http:
-        keystore_type: JKS
-        keystore_filepath: /etc/elasticsearch/secret/key
-        keystore_password: kspass
-        enforce_clientauth: true
-        truststore_type: JKS
-        truststore_filepath: /etc/elasticsearch/secret/truststore
-        truststore_password: tspass
-  actionrequestfilter:
-    names: ["readonly", "fluentd", "kibana", "curator", "admin"]
-    readonly:
-      allowed_actions: ["indices:data/read/*", "*monitor*"]
-      forbidden_actions: ["cluster:*", "indices:admin*"]
-    fluentd:
-      allowed_actions: ["indices:data/write/*", "indices:admin/create"]
-    kibana:
-      allowed_actions: ["indices:data/read/*", "*monitor*", "indices:admin/read", "indices:admin/mappings/fields/get*"]
-    curator:
-      allowed_actions: ["indices:admin/get", "indices:admin/delete"]
-
-openshift:
-  acl:
-    users:
-      names: ["system.logging.fluentd", "system.logging.kibana", "system.logging.curator", "system.admin"]
-      system.logging.fluentd:
-        execute: ["actionrequestfilter.fluentd"]
-        actionrequestfilter.fluentd.comment: "Fluentd can only write"
-      system.logging.kibana:
-        bypass: ["*"]
-        execute: ["actionrequestfilter.kibana"]
-        actionrequestfilter.kibana.comment: "Kibana can only read from every other index"
-      system.logging.kibana.*.comment: "Kibana can do anything in the kibana index"
-      system.logging.kibana.*.indices: [".kibana.*"]
-      system.logging.curator:
-        execute: ["actionrequestfilter.curator"]
-        actionrequestfilter.curator.comment: "Curator can list all indices and delete them"
-      system.admin:
-        bypass: ["*"]
-      system.admin.*.comment: "Admin user can do anything"
+      enabled: true
+      enforce_hostname_verification: false
+      enabled_protocols: ["TLSv1", "TLSv1.1", "TLSv1.2"]
```

Comment:
I think I was playing around with the TLS versions ES supports when Curator wasn't able to connect.  I'll retest without TLSv1 now that Curator is working